assets/admin/students/student-action-menu/index.js

```diff
@@ -68,7 +70,7 @@ export const StudentActionMenu = ( {
 	 *
 	 * @return {Array} Filtered controls.
 	 */
-	const controls = applyFilters(
+	applyFilters(
```

Comment:
Why / how does this code work? It shouldn't as I'm not calling `setControls` and the component doesn't re-render after the fetch in Sensei Pro returns. Even so, the "Add to Group" and "Remove from Group" options are added.